code/src/java/pcgen/rules/context/AbstractReferenceContext.java

```diff
@@ -127,13 +128,17 @@ public boolean validate(UnconstructedValidator validator)
 		return getManufacturer(c, cat).getTypeReference(val);
 	}
 
-	public <T extends Loadable> T constructCDOMObject(Class<T> c, String val)
+	public <T extends Loadable, U extends Loadable & CategorizedCDOMObject<U>> T constructCDOMObject(Class<T> c, String val)
```

Comment:
I have removed the "U" types from the file since they are confusing.  I am looking at other strategic changes to the context files, which should eliminate the need for that behavior, but is 6.7 behavior since it is dependent on AbilityCategory.FEAT not being "preloaded"